sdk/cosmos/azure-cosmos-test/CHANGELOG.md

```diff
@@ -7,8 +7,7 @@
 #### Breaking Changes
 
 #### Bugs Fixed
-
-#### Other Changes
+* Fixed an issue where `FaultInjectionRule` can not apply on partition level when using `Gateway` Mode and non-session consistency - See [40005](https://github.com/Azure/azure-sdk-for-java/pull/40005)
```

Comment:
Do we target this PR to be merged into - `Azure:feature/vector_search`? Also looks like we have some `CHANGELOG`s leaking into this PR which do not reflect what this PR adds.

Reply:
Fixed it.